target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8064-eax500.dtsi

```diff
@@ -0,0 +1,233 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	chosen {
+		bootargs = "console=ttyMSM0,115200n8";
+		append-rootblock = "ubi.mtd=";  /* append to bootargs adding the root deviceblock nbr from bootloader */
+	};
+
+	keys: keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&button_pins>;
```

Comment:
I don't have a strong opinion on this, but is it okay to use a label that is not defined in the dtsi?

Reply:
I considered moving part of the pinmux to the DTSI, but found that this will only make it less readable.
So, I've put the keys node back to the DTS files, there should be enough shared stuff beyond that.